src/components/Web3Status/index.tsx

```diff
@@ -272,6 +272,7 @@ function Web3StatusInner() {
       ...CHEVRON_PROPS,
       color: theme.accentAction,
       'data-testid': 'navbar-wallet-dropdown',
+      'aria-label': `${walletIsOpen ? 'close' : 'open'} wallet connection options`,
```

Comment:
These should be sentence-cased and translated. I think they should also be added to the chevron on L265:

    import { t, Trans } from '@lingui/macro'
    ...
    'aria-label': walletIsOpen ? t`Show wallet connection options` : t`Hide wallet connection options`,